radis/levels/partfunc.py

```diff
@@ -1146,6 +1151,57 @@ def _at(self, T):
         return np.interp(T, self.T_range, self.Q_range)
 
 
+class PartFuncKurucz(RovibParFuncTabulator):
+    """Return partition function using interpolation of tabulated values of local file kuruczpartfn.txt
+    """
+
+    def __init__(self, species, pfpath = None):
+        super(PartFuncKurucz, self).__init__()
+        # Load data in constructor
+        self.species = species
+        self.pfTdat, self.pfdat = load_pf_Barklem2016()
+        # Locate the row for the specific atom and ionization state
+        try:
+            pf_atom = self.pfdat.loc[f"{species}"]
+        except KeyError:
+            warn("Note: the partition functions from Barklem & Collet (2016) don't include this species.")
```

Comment:
raise an error on this. It is normally not possible to compute a spectrum without the partition function